slo_generator/backends/cloud_monitoring.py

```diff
@@ -180,15 +180,20 @@ def query(self,
         Returns:
             list: List of timeseries objects.
         """
+        if group_by is None:
+            group_by = []
         measurement_window = CM.get_window(timestamp, window)
         aggregation = CM.get_aggregation(window,
                                          aligner=aligner,
                                          reducer=reducer,
                                          group_by=group_by)
-        timeseries = self.client.list_time_series(
-            self.parent, filter, measurement_window,
-            monitoring_v3.enums.ListTimeSeriesRequest.TimeSeriesView.FULL,
-            aggregation)
+        request = monitoring_v3.ListTimeSeriesRequest()
```

Comment:
This is a good example of how I had to rewrite the existing code for `google-cloud-monitoring` v2.